net-dns/dnscrypt-proxy/dnscrypt-proxy-2.0.0.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGO_PN="github.com/jedisct1/dnscrypt-proxy/..."
+EGO_SRC="github.com/jedisct1/dnscrypt-proxy"
+
+inherit systemd user golang-build
+
+DESCRIPTION="A flexible DNS proxy, with support for modern encrypted DNS protocols."
+HOMEPAGE="https://${EGO_PN}"
+SRC_URI="https://${EGO_PN}/archive/2.0.0.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="systemd"
```

Comment:
A use flag to control 2 lines of ````elog```` in _pkg_postinst()_? Seems wrong.

Reply:
I started from the previous ebuild. The original ebuild requires the systemd dependency when systemd use is specified.